src/libs/blueprint/conduit_blueprint_mpi_mesh.cpp

```diff
@@ -1252,9 +1254,10 @@ void to_polyhedral(const Node &n,
                 const Node& group = grp_itr.next();
                 if (group.has_child("neighbors") && group.has_child("windows"))
                 {
-                    int64_array neighbors = group["neighbors"].as_int64_array();
-
-                    index_t nbr_id = neighbors[1];
+                    temp.reset();
```

Comment:
FYI: There is a new pattern you can use to simplify logic like this, you can use:

int64_accessor vals = group["neighbors"].value();

`vals`  will give you access as an int64 regardless of underlying type, and handle the offset, striding.

Reply:
Thanks for the comment, @cyrush !  this will be handy for some of my code as well.

Reply:
I'm going to leave this as it is, as I was trying to access everything as `index_t` rather than `int64`, but it had been missed in a few places in this file; this tip will be helpful for future coding.

Reply:
You can also use `index_t_accessor`.

Regardless - find to leave as is, I think we will do a large pass a pass in the future to simplify cases like this. 

Thanks again!